visiting.md

```diff
@@ -1,20 +1,60 @@
 ---
 layout: page
-order: 0
+order: 4
 title: "Visiting Toronto"
 titleDisplay: "Visiting"
+parent: "About the Event"
 ---
 
-We have a slowly-growing list of questions for those visiting Tkaronto (Toronto), if you don't see yours answered below, send us an email at [{{ site.email }}](mailto:{{ site.email }}).
+## Getting Around 
 
-### I am traveling from out of town, how can I stay in Toronto affordably? (a.k.a. What are community billets?)
+Toronto is easily navigable by bike ([Bike Share Toronto](https://bikesharetoronto.com/) as a popular rental option), using public transportation ([TTC](http://www.ttc.ca/)), and on foot around the event venue.
 
-We have arranged a limited number of spots with people based in Toronto for those looking for a place to stay. Billeting is a way of supporting participants to attend within their budget and at the same time, welcome them into our community. If you indicated you were interested in hosting or staying in a community billet we will be contacting you.
+## Staying in Toronto
 
-### What is the best way to get around Toronto?
+The city is an expensive place to live and visit. For those looking at alternative acccomodations including platforms like AirBnB, we wanted to provide context about the housing market. Like many cities around the world, Toronto is experiencing an [affordability crisis](https://www.blogto.com/city/2019/03/report-housing-affordability-crisis-toronto/) in part driven by [condos being treated as investments](https://www.theguardian.com/world/2019/jul/07/toronto-housing-owner-occupied-canada-affordability) and ["ghost hotels"](https://www.thestar.com/opinion/star-columnists/2017/12/01/haunted-by-the-rise-of-ghost-hotels.html). This has recently led to [renovictions](https://nowtoronto.com/news/renovicted-toronto-rental-housing/) and more recently [tenant organizing and rent strikes](http://parkdaleorganize.ca/).
```

Comment:
```suggestion
The city is an expensive place to live and visit. For those looking at alternative accommodations including platforms like AirBnB, we wanted to provide context about the housing market. Like many cities around the world, Toronto is experiencing an [affordability crisis](https://www.blogto.com/city/2019/03/report-housing-affordability-crisis-toronto/) in part driven by [condos being treated as investments](https://www.theguardian.com/world/2019/jul/07/toronto-housing-owner-occupied-canada-affordability) and ["ghost hotels"](https://www.thestar.com/opinion/star-columnists/2017/12/01/haunted-by-the-rise-of-ghost-hotels.html). This has recently led to [renovictions](https://nowtoronto.com/news/renovicted-toronto-rental-housing/) and more recently [tenant organizing and rent strikes](http://parkdaleorganize.ca/).
```

Reply:
I'm confused about this, I think this is fixed in my last commit